Src/BootCamp.Chapter/Lesson4.cs

```diff
@@ -0,0 +1,91 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace BootCamp.Chapter
+{
+    class Lesson4
+    {
+        public static void Demo()
+        {
+            for (int i = 0; i < 2; i++)
+            {
+                string name = PrintAndReadString("First Name: ");
+
+                string sureName = PrintAndReadString("Last Name: ");
+
+                int age = PrintAndReadInt("Age: "); ;
+
+                float weight = PrintAndReadFloat("Weight: ");
+
+                float height = PrintAndReadFloat("Height: ");
+
+                var bmi = BMICalc(weight, height);
+
+                Console.WriteLine($"{name} {sureName} is {age} years old, his weight is {weight} kg and his height is {height} cm.");
+
+                Console.WriteLine($"Your BMI - {bmi:N2}");
+            }
+        }
+
+        public static float BMICalc(float weight, float height)
```

Comment:
Don't short the name- wirte the full. 
Also if you switch places (CalculateBmi), this becomes a verb- that's what a method name should be based on (action)